handbook/product.md

```diff
@@ -4,6 +4,82 @@
 
 🧱📡 Fleet core roadmap board: https://github.com/orgs/fleetdm/projects/8
 
+## Job to be done
+
+Every product should have a single job that it strives to do. At Fleet we use the [Jobs to be Done
+(JTBD) framework](https://about.gitlab.com/handbook/engineering/ux/jobs-to-be-done/). Fleet's overarching job to be done is:
+
+> When we are maintaining secure laptops and servers as a team, we want to get from understanding
+> what's going on to knowing what we need to do with quality, reliability, and low impact quickly
+> and within budget, so that our organization can achieve our business outcomes.
```

Comment:
@noahtalerman To further simplify, we could try:

> "I need a way to see what laptops and servers I have, and what I need to do to keep them secure and compliant."

Reply:
I like this 👍  Makes it easier for the job to morph / grow later.